### DISCUSSION TITLE
get_property error

### DISCUSSION THREAD

**Discussion by indirabarros** [DeepTrackAI/DeepTrack2#225] [Q&A]
I am trying to run the tutorials available on the page, but i keep getting an error saying "'numpy.ndarray' object has no attribute 'get_property'". How do I resolve this? I tried using "pipeline.store_properties()", but that does not seem to work either. I am new here and would be grateful for any help. Thank you.

**Comment by BenjaminMidtvedt** [comment]
Hi! @indirabarros Are you using LodeSTAR? If so, this is a known bug.
Either,
1. Downgrade to 1.7
2. Pull the latest version from develop and use that
